src/olympia/migrations/1212-unique-lowercase-usernames.sql

```diff
@@ -0,0 +1,11 @@
+/* Create a table with all usernames that have duplicates in lowercase. */
+CREATE TABLE `duplicate_usernames`
```

Comment:
Can we either delete this table at the end of the migration, or file a follow-up to remove it once we're done with whatever we're doing with it?

Reply:
or is `CREATE TEMPORARY TABLE` a possibility?  (Is the entire migration executed as a single "session")?

Reply:
I almost did that but I thought we might want to have it around in case we receive some user complaints... though I realize now I'm just storing the username and not the id so I don't know how useful it'd be...

Reply:
And the ```LOWER(`username`)``` too, so even less useful.  If you want to to keep it around for some reason then issue plz.  Now we've a clean database lets keep it clean 😛

Reply:
23584554f80007cae97061c49f26e72dcf7f59c1 makes it a temporary table